boards/rcar_salvator_xs_m3_2x4g.overlay

```diff
@@ -48,3 +43,7 @@
 		};
 	};
 };
+
+&ram {
+	reg = <0x00 0x60000000 0x00 DT_SIZE_M(256)>;
```

Comment:
(note) as far as I know we have the same config for this board inside overlay for xenvm dom0